subsys/bluetooth/host/hci_ecc.c

```diff
@@ -347,13 +347,15 @@ void bt_hci_ecc_supported_commands(uint8_t *supported_commands)
 	supported_commands[41] |= BIT(2);
 }
 
-int default_CSPRNG(uint8_t *dst, unsigned int len)
+static int ecc_csprng(uint8_t *dst, unsigned int len)
 {
 	return !bt_rand(dst, len);
 }
 
 void bt_hci_ecc_init(void)
 {
+	uECC_set_rgn(&ecc_csprng);
```

Comment:
typo...  uECC_set_rng()

also, as this configures global, could affect system in unexpected way